frida_mode/include/complog.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _COMPLOG_H
```

Comment:
In Frida all occurences of "CompLog" should be renamed to our previously used "CmpLog" to avoid confusion, please!
That is: in files, functions and documentation

Reply:
good catch, removed files and renamed files etc.